### PR TITLE
Add a notebook to demonstrate how to run notebooks from another notebook

### PR DESCRIPTION
**Pull request by nunogoncalves03**
cc: @ricardoasmarques @kanitsharma

### CODE REVIEW

File: notebooks/executing-notebooks-from-another-notebook-with-fusion-sql/notebook.ipynb [hunk 1]
```diff
@@ -0,0 +1,306 @@
...
+        "print(f'All jobs completed with success: {bool(success[0].Success)}')"
+      ],
+      "id": "2cdaea66"
+    },
+    {
+      "attachments": {},
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "View the executions of the jobs we run:"
```
Comment (ricardoasmarques) [inline notebooks/executing-notebooks-from-another-notebook-with-fusion-sql/notebook.ipynb:209]: `run` -> `ran`

**Review by kesmit13 — COMMENTED** [review]
I'm wondering if we should create new magic commands called `%run_shared` and `%run_personal` that are thin wrappers around `%run` and do the download in the background. We just published a `%sqlx` magic in the last release of the Python SDK to allow `%sql`-like behaviors in multi-threaded environments, so there's a precedent now.

File: notebooks/executing-notebooks-from-another-notebook-with-fusion-sql/notebook.ipynb (hunk 1 above)
Comment (nunogoncalves03) [reply notebooks/executing-notebooks-from-another-notebook-with-fusion-sql/notebook.ipynb]: Thank you for pointing out that typo; I have already fixed it!

**Comment by nunogoncalves03** [comment]
> I'm wondering if we should create new magic commands called `%run_shared` and `%run_personal` that are thin wrappers around `%run` and do the download in the background. We just published a `%sqlx` magic in the last release of the Python SDK to allow `%sql`-like behaviors in multi-threaded environments, so there's a precedent now.

That's a pretty interesting idea. It's definitely something we should consider, thanks for the suggestion! What do you think @ricardoasmarques?

File: notebooks/executing-notebooks-from-another-notebook-with-fusion-sql/meta.toml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+[meta]
+authors=["singlestore"]
+title="Executing Notebooks from Another Notebook with Fusion SQL"
```
Comment (ricardoasmarques) [inline notebooks/executing-notebooks-from-another-notebook-with-fusion-sql/meta.toml:3]: Maybe rename all occurrences of `execute/executing` to `run/running` for consistency with the actions in the UI and `%run` magic command (including folder name).

**Comment by ricardoasmarques** [comment]
> > I'm wondering if we should create new magic commands called `%run_shared` and `%run_personal` that are thin wrappers around `%run` and do the download in the background. We just published a `%sqlx` magic in the last release of the Python SDK to allow `%sql`-like behaviors in multi-threaded environments, so there's a precedent now.
> 
> That's a pretty interesting idea. It's definitely something we should consider, thanks for the suggestion! What do you think @ricardoasmarques?

Yes, I really like the idea. Let's create a ticket to track that and update this Notebook accordingly once it is implemented.

**Pull request merged**